integrations/appboy/lib/index.js

```diff
@@ -43,32 +45,25 @@ var Appboy = (module.exports = integration('Appboy')
     '<script src="https://js.appboycdn.com/web-sdk/2.4/appboy.min.js">'
   ));
 
-Appboy.prototype.initialize = function() {
-  var options = this.options;
-  var customEndpoint;
-  // Setup custom endpoints
-  if (options.customEndpoint) {
```

Comment:
Custom endpoint gets initialized later. No need to initialize it here.

Reply:
This helped clean up the code a little to make it easier to add hooks for delaying initialization.